docs/assembly.rst

```diff
@@ -334,38 +341,39 @@ Literals
 You can use integer constants by typing them in decimal or hexadecimal notation and an
 appropriate ``PUSHi`` instruction will automatically be generated. The following creates code
 to add 2 and 3 resulting in 5 and then computes the bitwise and with the string "abc".
```

Comment:
Maybe combine both sentences? "...resulting in 5, then computes the bitwise ``and`` with the string "abc" and assignes the final value to a local variable called ``x``"